src/dorsal.js

```diff
@@ -160,7 +160,11 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
```

Comment:
instead of doing this in here. would be better to have always this.plugins = {} instead of having the eval on 
```DorsalCore.prototype.registerPlugin = function(pluginName, callback) {
    if (!this.plugins) {
        this.plugins = {};
    }
    this.plugins[pluginName] = callback;
};
```
DorsalCore.prototype.plugins = {};
```
so this code could remain the same, with out this logic.